index.html

```diff
@@ -5,13 +5,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
 </head>
 <body>
     <input type="text" id="foods-input">
     <button id="btn-add-foods">Add</button>  
     <table id="foods-table"></table>
 
 
-<script src="./main.js"></script>
+    <script src="./main.js"></script>
```

Comment:
Dont break line .

Reply:
The fuk was that, are u kidding me, i dont think so